src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+
+  notification.innerHTML = `
+    <h2 class="title">${title}</h2>
+    <p>${description}</p>
+  `;
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
```

Comment:
not blocker: would be better to go ahead with 1 way to do same things. in this case we are talking about working with new elements. Choose your way to do it, with html in string or with `createElement`